project/AppDependencies.scala

```diff
@@ -17,7 +17,7 @@ object AppDependencies {
 
   val test = Seq(
     "org.scalatest"              %% "scalatest"                 % "3.0.8"          % "test",
-    "org.jsoup"                  % "jsoup"                      % "1.10.2"         % "test",
+    "org.jsoup"                  % "jsoup"                      % "1.8.3"         % "test",
```

Comment:
is there a reason we can't jump to the latest?

Reply:
done, it was some kind of mistake :) Good spot, thanks